bp_be/test/tb/bp_be_dcache/py/trace_script.py

```diff
@@ -0,0 +1,134 @@
+#!/bin/usr/python

```

Comment:
Like the bsg_cache testbench, separate out each test into a script to generate a rom (or have 1 script that generates multiple roms, your choice)

Reply:
Yeah, we're planning to have the same script but generate different traces for each block of tests.